docs/setup/settings.asciidoc

```diff
@@ -339,12 +361,34 @@ https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT[OpenS
 connections. Valid values are `required`, `optional`, and `none`. `required` forces a client to present a certificate, while `optional`
 requests a client certificate but the client is not required to present one.
 
-`server.ssl.enabled:`:: *Default: "false"* Enables SSL for outgoing requests
-from the Kibana server to the browser. When set to `true`,
-`server.ssl.certificate` and `server.ssl.key` are required.
+`server.ssl.enabled:`:: *Default: "false"* Enables SSL for inbound requests from the browser to the Kibana server. When set to `true`, a
+certificate and private key must be provided. These can be specified via `server.ssl.keystore.path`, or the combination of
+`server.ssl.certificate` and `server.ssl.key`.
+
+`server.ssl.keyPassphrase:`:: The passphrase that will be used to decrypt the private key that is specified via `server.ssl.key`. This value
+is optional, as the key may not be encrypted.
+
+`server.ssl.keystore.path:`:: Path to a PKCS #12 file -- this should contain an X.509 certificate with its private key. These are used when
+enabling SSL for inbound requests from web browsers to the Kibana server. If the file contains any additional certificates, those will be
+used as a trusted certificate chain for Kibana. This is used when a web browser creates an SSL connection with the Kibana server; the
+certificate chain is sent to the browser along with the end-entity certificate to establish trust. This is also used to determine if client
```

Comment:
```suggestion
certificate chain is sent to the browser along with the end-entity certificate to establish trust. This setting is also used to determine whether client
```

Reply:
I meant "this chain" here, will qualify it accordingly.